packages/fast-breakpoint-tracker-react/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@microsoft/fast-breakpoint-tracker-react",
+  "description": "React component for managing breakpoints",
+  "sideEffects": false,
+  "version": "1.0.0",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast-dna.git"
+  },
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json",
+    "clean:dist": "node ../../build/clean.js dist",
+    "prepare": "npm run clean:dist && npm run build",
+    "prettier": "prettier --config ../../.prettierrc --write '**/*.{ts,tsx}'",
+    "prettier:diff": "prettier --config ../../.prettierrc '**/*.{ts,tsx}' --list-different",
+    "test": "npm run tslint && npm run unit-tests",
+    "tslint": "tslint -c tslint.json '**/*.ts{,x}'",
+    "tslint:fix": "tslint -c tslint.json '**/*.ts{,x}' --fix",
+    "tslint:watch": "node build/tslint-watch.js",
+    "unit-tests": "jest --maxWorkers=4",
+    "unit-tests:watch": "jest --watch && npm-watch",
+    "watch": "npm run build -- -w --preserveWatchOutput"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "coverageThreshold": {
+      "global": {
+        "statements": 80,
+        "branches": 50,
```

Comment:
Any way to up this coverage to 80?

Reply:
Currently there is no way to valuable way to test the resize event of the window that the component should respond to. While I could create brittle fake tests to up the coverage, they wouldn't be representative of what/how we should test and would serve only to help the branch coverage look good